src/site/content/en/lighthouse-best-practices/image-aspect-ratio/index.md

```diff
@@ -17,8 +17,8 @@ possibly creating an unpleasant user experience.
 ## How the Lighthouse image aspect ratio audit fails
 
 [Lighthouse](https://developers.google.com/web/tools/lighthouse/) flags any
-image with a rendered aspect ratio
-5 percent or more different from its natural ratio:
+image with a rendered dimension more than a few pixels different from the expected dimension
```

Comment:
my natural inclination was to be a bit terser here, "a rendered dimensions more than a few pixels different than its expected size", but then I couldn't justify why have a doc about it so went for the more verbose option :)

suggestions welcome